config/main

```diff
@@ -0,0 +1 @@
+include config/default
```

Comment:
For v6_images I don't know, but for the UI and API ``config/main`` is the config file used for the demo instance.